src/core/lib/security/credentials/tls/grpc_tls_certificate_provider.cc

```diff
@@ -21,11 +21,13 @@
 #include <grpc/support/alloc.h>
 #include <grpc/support/log.h>
 #include <grpc/support/string_util.h>
+#include <openssl/base.h>
```

Comment:
This won't work with OpenSSL. You can also omit it because all our other headers include it in BoringSSL.

Reply:
Thanks, noted.